test/unit/eigensolver/test_band_to_tridiag.cpp

```diff
@@ -157,24 +159,29 @@ void testBandToTridiag(CommunicatorGrid grid, blas::Uplo uplo, const SizeType ba
 TYPED_TEST(EigensolverBandToTridiagTest, CorrectnessLocalFromCPU) {
   const blas::Uplo uplo = blas::Uplo::Lower;
 
-  for (const auto& [m, mb, b] : sizes)
+  for (const auto& [m, mb, mb_1d, b] : sizes) {
+    getTuneParameters().band_to_tridiag_1d_block_size_base = mb_1d;
```

Comment:
For the local implementation it doesn't have any effect. I modified the parameter anyway, let me know if you prefer to not touch it.

Reply:
Fine by me. Would it ever be useful to make it have an effect for the local implementation?

Reply:
No, for local matrices the behaviour is similar as if the tuning value equal to the matrix size.